CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 Change log
 ==========
 
+1.5.0 (2015-10-13)
+------------------
+
+Major Features
+
+-   Compose is now available on windows.
```

Comment:
If https://github.com/docker/compose/pull/2181 makes it into the release candidate, you can ignore my comment above.